test/extensions/voting-rep.js

```diff
@@ -332,6 +332,13 @@ contract("Voting Reputation", (accounts) => {
       expect(revealPeriod).to.eq.BN(REVEAL_PERIOD);
       expect(escalationPeriod).to.eq.BN(ESCALATION_PERIOD);
     });
+
+    it("can't use the network-level functions if installed via ColonyNetwork", async () => {
+      // await checkErrorRevert(voting.install(ADDRESS_ZERO, { from: USER1 }), "ds-auth-unauthorized");
```

Comment:
Because the voting contract doesn't fail that way. Wanted to bring this up so we could explicitly consider whether we're okay with this or not.

Reply:
Was there a reason we/I did this? Seems not strictly necessary...

Reply:
I don't think there was a reason, no. It would be better if it was auth'd though, as in the scenario where someone is deploying it independently, as-is someone could watch for the code being deployed and grief by calling the function.